backend/src/test/resources/tests/filter/POSTAL_CODE/expected_POSTAL_CODE_Filter.csv

```diff
@@ -0,0 +1,3 @@
+result,dates
+2,{2016-02-01/2016-02-01}
+4,{2019-04-01/2019-04-01}
```

Comment:
Kann es sein, das der Test noch nicht klappt? Warum ist pid 1 nicht vorhanden, aber 2 und 4:
| plz1 | plz2 | Distance_convert_in_km |
| --- | --- | --- |
| 10115 |  10249 | 39.788 | 
| 10115  | 10405 | 27.014 |


Generell finde ich für die PLZ 10115 nur 34 Einträge in der plz-csv-Datei

Reply:
die pid 1 ist nicht da, weil ich bei der letzten Implementierung die Referenz-PLZ (für den Fall 10115) ausgeschlossen habe

Reply:
Okay, aber die anderen dürften nicht drinne sein, da sie nicht im Radius von 20 km liegen

Reply:
Warum ist die Referenz PLZ ausgeschlossen?

Reply:
weil man halt nach den PLZs sucht, die innerhalb eines Radius sind und nicht die eigene PLZ

Reply:
Das schließt eben diese PLZ doch ein :)